Deploy-CAM.ps1

```diff
@@ -1813,6 +1860,9 @@ function Deploy-CAM() {
 
         [parameter(Mandatory = $true)]
         $vnetConfig,
+
+        [parameter(Mandatory=$false)]
```

Comment:
I think it's mandatory or you will have de-referencing null object issues in the function.

Reply:
You're right, easier/safer if I just make this mandatory